MonkeyWrench.Web.UI/ViewLane.aspx.cs

```diff
@@ -256,7 +256,7 @@ public string GenerateLane (GetViewLaneDataResponse response)
 			} else {
 				if (response.RevisionWork.State == DBState.NotDone)
 					header.AppendFormat (" - <a href='ViewLane.aspx?lane_id={0}&amp;host_id={2}&amp;revision_id={1}&amp;action=ignorerevision'>don't build</a>", lane.id, dbr.id, host.id);
-				if (response.RevisionWork.State != DBState.NoWorkYet) {
+				if (response.RevisionWork.State == DBState.Failed || (response.RevisionWork.State != DBState.NoWorkYet && lane.priority != 2)) {
```

Comment:
This will prevent `reset work` from showing up for builds with test failures, is that intentional?